### DISCUSSION TITLE
Saj H1, more info about battery

### DISCUSSION THREAD

**Discussion by TecnoTalarn** [djansen1987/SAJeSolar#38] [Ideas]
I use this tool and I'm happy, but I not see 2, I think, important values.
1st is the voltage on my battery. 
2nd is the  battery temperature.

The voltage so sometimes I see BMS communication errors and the voltage increases until an alarm is generated and the inverter shuts down.
it almost never happens, but it does happen. It would be interesting to be able to anticipate and from the HA send an alert or take some action to reduce the voltage, such as consuming power.
The temperature is also interesting as it gives us a lot of information about possible problems with the battery. And it gives us security, especially if, like most of us, we are out of the house for many hours a day.

I don't know if it would be possible to read this data, but I see that many APPs provide it, so they must be in the SAJ cloud.

Finally, an interesting piece of information would be to be able to see each one of the MPPT separately, but this is more of a statistical issue, since I have them in different orientations and I would like to see which one gives me the best performance.

**Comment by djansen1987** [comment]
Hi, that sounds indeed important. Do you know if those values are in the portal, and if they are where can i find them? Maybe then i can see if i can find a matching API endpoint to get the sensor data out of.
